### PR TITLE
 Review API Proxy tutorial

### PR DESCRIPTION
**Pull request by shaniR**

### CODE REVIEW

File: en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md [hunk 1]
```diff
@@ -33,20 +34,23 @@ In this tutorial, you will be importing an OpenAPI specification and creating a
     !!! tip
         Once you create a project successfully, you are taken to the components landing page. You can also select the components icon from the left menu and open the **Components** landing page. 
 
-4. Click the **REST API Proxy** card.
+4. Click **+Create** on the **Components** landing page.
```
Comment (RukshiW) [inline en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md:37]:
```suggestion
4. On the **Components** landing page, click **Create**.
```

Suggesting this to adhere to this guideline in the Google Development guide:

https://developers.google.com/style/procedures#steps-that-say-where-to-complete-a-task

File: en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md [hunk 2]
```diff
@@ -16,13 +16,14 @@ Let's begin...
 
 Choreo gives you two options to create a REST API proxy:
```
Comment (RukshiW) [inline en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md:17]: The word "methods" seem more correct here than "options".

```suggestion
You can create a REST API proxy iin one of the following methods:
```

File: en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-# Create Your First REST API Proxy
+# Expose an existing API via a REST API proxy
 
 Choreo is a low-code cloud-native platform for building integration solutions. In this tutorial, you will explore REST API Proxies in Choreo by:
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md]: Done

File: en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md (hunk 2 above)
Comment (shaniR) [reply en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md]: done

**Review by RukshiW — APPROVED** [review]

**Comment by RukshiW** [comment]
Did some minor changes. Refer the Commit descriptions for more details.

This tutorial also needs to be updated with the latest UI changes. Reported this in a separate issue [1].

[1] https://github.com/wso2-enterprise/choreo/issues/14778

**Comment by shaniR** [comment]
Will be tracked by https://github.com/wso2/docs-choreo-dev/pull/599

**Pull request closed without merging**